packages/client/src/pages/location/[locationName].tsx

```diff
@@ -27,6 +29,7 @@ const LocationPage = () => {
 
   const params = match.params as any
   const locationName = params.locationName ?? `${params.projectName}/${params.sceneName}`
+  const oembedLink = `${serverHost}/oembed?url=${encodeURIComponent(`${clientHost}/location/${locationName}`)}&format=json`
```

Comment:
this should be implemented in _app.tsx

i moved the other metadata (coil, title etc) to _app recently, this is the proper place for it. putting it in the Layout component will only show up for locations

Reply:
We should have a quick discussion about how each type of page should be oembedded, then. Locations we've covered, home page was sort of covered, just want to confirm what we want for editor and admin, and any non-default routes.

Reply:
Handled by all non-location routes just returning a plain link to that page.